scripts/link_linter.js

```diff
@@ -0,0 +1,191 @@
+const parse = require("node-html-parser");
```

Comment:
Adding a lint rule to enforce that we are setting the newly added data links on all `<a>` tags? I wear a single emoji tear of joy. 😢 😀